Altis_Life.Altis/config/Config_Loadouts.hpp

```diff
@@ -2,77 +2,118 @@
+            uniformClass = "";
+            backpack = "";
+            linkedItems[] = {};
+            weapons[] = {};
+            items[] = {};
+            magazines[] = {};
         };
     };
 */
+class lvl_base {
+    uniformClass = "";
+    backpack = "";
+    linkedItems[] = {"ItemMap","ItemCompass","ItemWatch"};
+    weapons[] = {};
+    items[] = {};
+    magazines[] = {};
+};
+
 class Loadouts {
-    // COP
-    class WEST {
-        uniform[] = {
-            {"U_Rangemaster", "call life_copLevel >= 0"}
+    // CIV
+    class CIV {
+        class lvl_arrested : lvl_base {
+            uniformClass = "U_C_WorkerCoveralls"; // Arrested player's uniform
+            linkedItems[] = {};
         };
-        vest[] = {
-            {"V_Rangemaster_belt", "call life_copLevel >= 0"}
+        class lvl_1 : lvl_base {
+            uniformClass = "U_C_Poloshirt_blue";
         };
-        backpack[] = {};
-        weapon[] = {
-            {"hgun_P07_snds_F", "call life_copLevel >= 0"}
+        class lvl_2 : lvl_base {
+            uniformClass = "U_C_Poloshirt_burgundy";
         };
-        mags[] = {
-            {"16Rnd_9x21_Mag", 6, "call life_copLevel >= 0"}
+        class lvl_3 : lvl_base {
+            uniformClass = "U_C_Poloshirt_stripped";
         };
-        items[] = {};
-        linkedItems[] = {
-            {"ItemMap", "call life_copLevel >= 0"},
-            {"ItemCompass", "call life_copLevel >= 0"},
-            {"ItemWatch", "call life_copLevel >= 0"}
+        class lvl_4 : lvl_base {
+            uniformClass = "U_C_Poloshirt_tricolour";
+        };
+        class lvl_5 : lvl_base {
+            uniformClass = "U_C_Poloshirt_salmon";
+        };
+        class lvl_6 : lvl_base {
+            uniformClass = "U_C_Poloshirt_redwhite";
+        };
+        class lvl_7 : lvl_base {
+            uniformClass = "U_C_Commoner1_1";
         };
     };
 
-    // CIV
-    class CIV {
-        uniform[] = {
-            {"U_C_Poloshirt_blue", "!life_is_arrested"},
-            {"U_C_Poloshirt_burgundy", "!life_is_arrested"},
-            {"U_C_Poloshirt_stripped", "!life_is_arrested"},
-            {"U_C_Poloshirt_tricolour", "!life_is_arrested"},
-            {"U_C_Poloshirt_salmon", "!life_is_arrested"},
-            {"U_C_Poloshirt_redwhite", "!life_is_arrested"},
-            {"U_C_Commoner1_1", "!life_is_arrested"}
-        };
-        vest[] = {};
-        backpack[] = {};
-        weapon[] = {};
-        mags[] = {};
-        items[] = {};
-        linkedItems[] = {
-            {"ItemMap", ""},
-            {"ItemCompass", ""},
-            {"ItemWatch", ""}
+    // COP
+    class WEST {
+        class lvl_1 : lvl_base {
+            uniformClass = "U_Rangemaster";
+            weapons[] = {"hgun_P07_snds_F"};
+            magazines[] = {"16Rnd_9x21_Mag","16Rnd_9x21_Mag","16Rnd_9x21_Mag","16Rnd_9x21_Mag","16Rnd_9x21_Mag","16Rnd_9x21_Mag"};
+        };
+        class lvl_2 : lvl_base {
```

Comment:
Its good to see a base class being used, however you could be using the first class to avoid repeating yourself below, especially when all loadouts have the same.
```suggestion
        class lvl_2 : lvl_1 {};
```